React/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,37 @@
+import { isArray } from './utils';
+
+export function min(...args) {
+  if (args.length === 0) {
+    return undefined;
+  }
+  let spread = [];
+  args.forEach(pos => {
+    if (isArray(pos)) {
+      spread = [...spread, ...pos];
+    } else {
+      spread.push(pos);
+    }
+  });
+  return Math.min(...spread);
+}
+
+export function copy(element) {
+  /*
```

Comment:
Remove this comment haha

Reply:
I knew I was going to forget to erase that xD